recipe/meta.yaml

```
@@ -17,14 +17,14 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.10
     - pip
```

```suggestion
    - pip <24 # upperbound until https://github.com/conda-forge/cgen-feedstock/issues/11 is closed
```

@erikvansebille could you apply the following? My permissions are tied to this PR (codeowners file) so i don't have the permissions. Related https://github.com/conda-forge/cgen-feedstock/issues/11